measurement/plugins/download_speed/measurements.py

```diff
@@ -106,88 +91,12 @@ def _find_least_latent_url(self, urls):
         initial_latency_results = []
         for url in urls:
             host = urlparse(url).netloc
-            initial_latency_results.append((url, (self._get_latency_results(host, 2))))
+            initial_latency_results.append((url, self._get_latency_results(host, 2)))
```

Comment:
I can't work out why this changed.

Reply:
Formatting change only. It would have been while running black